cmd/format-xl.go

```diff
@@ -209,6 +243,49 @@ func formatXLMigrateV1ToV2(export string) error {
 	return ioutil.WriteFile(formatPath, b, 0644)
 }
 
+func formatXLMigrateV2ToV3(export string) error {
```

Comment:
can you add comments and some unit tests for this?

Reply:
We don't have unit tests for formatXLMigrateV1ToV2 too, You think we need unit tests for individual format migration functions? We don't have unit tests for individual functions which migrate config.json too. Since we don't touch the migration code once written it might be OK to not have individual tests as we have `TestFormatXLMigrate` which tests for migration of versions from v1 to v3.

Reply:
oh yes understood we could simply validate version number after migration.